gpii/node_modules/flowManager/src/UserLogonStateChange.js

```diff
@@ -115,49 +137,77 @@ fluid.defaults("gpii.flowManager.userLogonStateChange.matchMakingStateChangeHand
 });
 
 
-// Incomplete grade - local FlowManager adds "gpii.flowManager.userLogonStateChange.matchMakingStateChangeHandler",
+// Incomplete grade - local FlowManager adds "gpii.flowManager.userLogonHandling.matchMakingStateChangeHandler",
 // whereas untrusted adds "gpii.flowManager.untrusted.stateChangeHandler"
 
-fluid.defaults("gpii.flowManager.userLogonStateChange.handler", {
+fluid.defaults("gpii.flowManager.userLogonHandling.handler", {
```

Comment:
This name isn't correct - proximityTrigger should appear in the final path segment